src/iptvsimple/PlaylistLoader.cpp

```diff
@@ -196,6 +196,8 @@ std::string PlaylistLoader::ParseIntoChannel(const std::string& line, Channel& c
     std::string strTvgShift   = ReadMarkerValue(infoLine, TVG_INFO_SHIFT_MARKER);
     std::string strCatchup       = ReadMarkerValue(infoLine, CATCHUP);
     std::string strCatchupDays   = ReadMarkerValue(infoLine, CATCHUP_DAYS);
+    if (strCatchupDays.empty())
+      strCatchupDays = ReadMarkerValue(infoLine, TVG_INFO_REC);
```

Comment:
Please just read the value into its own variable here and then add any logic here: https://github.com/kodi-pvr/pvr.iptvsimple/blob/98302028dd54789b3b0f2d134f09d49284e9c8e4/src/iptvsimple/PlaylistLoader.cpp#L259-L269

Otherwise the logic around catchup days will be scattered. Better to have it in one place.